tools/system_libs.py

```diff
@@ -407,6 +408,9 @@ def build_with_ninja(self, build_dir, libname):
     utils.safe_ensure_dirs(build_dir)
 
     cflags = self.get_cflags()
+    if self.deterministic_paths:
+      source_dir = utils.path_from_root()
+      cflags += [f'-ffile-prefix-map={source_dir}=/emsdk/emscripten', f'-fdebug-compilation-dir=/emsdk/emscripten']
```

Comment:
How about just `/emscripten/..` since this is not really emsdk related?

Reply:
Sure, I could go either way. Practically speaking this will only end up showing up for emsdk users, right? Do users other than us use embuilder?

Reply:
I don't feel strongly either.  lgtm either way.